tests/backup/backup_upgrade_test.go

```diff
@@ -204,3 +205,366 @@ var _ = Describe("{StorkUpgradeWithBackup}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudAccountName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{UpgradeScenario}", func() {
```

Comment:
Add doc string for the testcase

Reply:
I will add the doc string once the TestRail is updated.

Reply:
Done. "This testcase executes and validates end-to-end backup and restore operations with PX-Backup upgrade"